### DISCUSSION TITLE
Newly supported device still shown as unsupported

### DISCUSSION THREAD

**Discussion by dhanukapulkit** [Koenkk/zigbee2mqtt#13864] [Devices]
Hi, 
I added a new Ledvance dimmable lamp last week and is now integrated in the Zigbee Herdsman Converters as well but it still shows that the device is unsupported. What am i doing wrong here?

Here are the logs:

Zigbee2MQTT:info  2022-09-05 08:19:18: Successfully interviewed '0xf0d1b8be240807e1', device has successfully been paired
Zigbee2MQTT:warn  2022-09-05 08:19:18: Device '0xf0d1b8be240807e1' with Zigbee model 'A60 DIM T' and manufacturer name 'LEDVANCE' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Zigbee2MQTT:info  2022-09-05 08:19:18: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":null,"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1","status":"successful","supported":false},"type":"device_interview"}'
Zigbee2MQTT:info  2022-09-05 08:19:18: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"0xf0d1b8be240807e1","supported":false},"type":"pairing"}'
Zigbee2MQTT:debug 2022-09-05 08:19:38: Received MQTT message on 'zigbee2mqtt/bridge/request/permit_join' with data '{"device":null,"time":254,"transaction":"qjwss-1","value":false}'
Zigbee2MQTT:info  2022-09-05 08:19:38: Zigbee: disabling joining new devices.
Zigbee2MQTT:info  2022-09-05 08:19:38: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":false},"status":"ok","transaction":"qjwss-1"}'
Zigbee2MQTT:debug 2022-09-05 08:19:45: Received MQTT message on 'zigbee2mqtt/bridge/request/device/remove' with data '{"block":false,"force":false,"id":"0xf0d1b8be240807e1","transaction":"8fxk4-1"}'
Zigbee2MQTT:info  2022-09-05 08:19:45: Removing device '0xf0d1b8be240807e1' (block: false, force: false)
Zigbee2MQTT:debug 2022-09-05 08:19:45: Clearing Home Assistant discovery topic for '0xf0d1b8be240807e1'
Zigbee2MQTT:info  2022-09-05 08:19:45: MQTT publish: topic 'zigbee2mqtt/0xf0d1b8be240807e1', payload ''
Zigbee2MQTT:info  2022-09-05 08:19:45: Successfully removed device '0xf0d1b8be240807e1' (block: false, force: false)
Zigbee2MQTT:info  2022-09-05 08:19:45: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/remove', payload '{"data":{"block":false,"force":false,"id":"0xf0d1b8be240807e1"},"status":"ok","transaction":"8fxk4-1"}'
Zigbee2MQTT:warn  2022-09-05 08:19:45: Device '0xf0d1b8be240807e1' left the network
Zigbee2MQTT:info  2022-09-05 08:19:45: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1"},"type":"device_leave"}'
Zigbee2MQTT:info  2022-09-05 08:19:45: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0xf0d1b8be240807e1"},"type":"device_removed"}'
Zigbee2MQTT:debug 2022-09-05 08:19:50: Received MQTT message on 'zigbee2mqtt/bridge/request/permit_join' with data '{"device":null,"time":254,"transaction":"8fxk4-2","value":true}'
Zigbee2MQTT:info  2022-09-05 08:19:50: Zigbee: allowing new devices to join.
Zigbee2MQTT:info  2022-09-05 08:19:50: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"8fxk4-2"}'
Zigbee2MQTT:info  2022-09-05 08:19:55: Accepting joining not in blocklist device '0xf0d1b8be240807e1'
Zigbee2MQTT:info  2022-09-05 08:19:55: Device '0xf0d1b8be240807e1' joined
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1"},"type":"device_joined"}'
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0xf0d1b8be240807e1"},"type":"device_connected"}'
Zigbee2MQTT:info  2022-09-05 08:19:55: Starting interview of '0xf0d1b8be240807e1'
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1","status":"started"},"type":"device_interview"}'
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0xf0d1b8be240807e1"},"type":"pairing"}'
Zigbee2MQTT:debug 2022-09-05 08:19:55: Device '0xf0d1b8be240807e1' announced itself
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1"},"type":"device_announce"}'
Zigbee2MQTT:info  2022-09-05 08:19:55: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0xf0d1b8be240807e1"},"type":"device_announced"}'
Zigbee2MQTT:debug 2022-09-05 08:19:56: Received Zigbee message from '0xf0d1b8be240807e1', type 'readResponse', cluster 'genBasic', data '{"appVersion":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2022-09-05 08:19:56: Skipping message, definition is undefined and still interviewing
Zigbee2MQTT:debug 2022-09-05 08:19:56: Received Zigbee message from '0xf0d1b8be240807e1', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2022-09-05 08:19:56: Skipping message, definition is undefined and still interviewing
Zigbee2MQTT:info  2022-09-05 08:19:56: Successfully interviewed '0xf0d1b8be240807e1', device has successfully been paired
Zigbee2MQTT:warn  2022-09-05 08:19:56: Device '0xf0d1b8be240807e1' with Zigbee model 'A60 DIM T' and manufacturer name 'LEDVANCE' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Zigbee2MQTT:info  2022-09-05 08:19:56: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":null,"friendly_name":"0xf0d1b8be240807e1","ieee_address":"0xf0d1b8be240807e1","status":"successful","supported":false},"type":"device_interview"}'
Zigbee2MQTT:info  2022-09-05 08:19:56: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"0xf0d1b8be240807e1","supported":false},"type":"pairing"}'
Zigbee2MQTT:debug 2022-09-05 08:20:00: Received MQTT message on 'zigbee2mqtt/bridge/request/permit_join' with data '{"device":null,"time":254,"transaction":"8fxk4-3","value":false}'
Zigbee2MQTT:info  2022-09-05 08:20:00: Zigbee: disabling joining new devices.
Zigbee2MQTT:info  2022-09-05 08:20:00: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":false},"status":"ok","transaction":"8fxk4-3"}'
Zigbee2MQTT:debug 2022-09-05 08:24:12: Saving state to file C:\GitHub\zigbee2mqtt\data\state.json
Zigbee2MQTT:debug 2022-09-05 08:24:57: Received Zigbee message from '0xf0d1b8be240807e1', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":52442720,"imageType":205,"manufacturerCode":4489}' from endpoint 1 with groupID 0
Zigbee2MQTT:warn  2022-09-05 08:24:57: Received message from unsupported device with Zigbee model 'A60 DIM T' and manufacturer name 'LEDVANCE'
Zigbee2MQTT:warn  2022-09-05 08:24:57: Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

**Comment by dhanukapulkit** [comment]
https://www.zigbee2mqtt.io/devices/4058075728981.html

This is the device.

**Reply by Koenkk** [reply]
The model number is wrong; `CLA 60 DIM` was added but yours has `A60 DIM T`, can you make a pr to fix this?

https://github.com/Koenkk/zigbee-herdsman-converters/blob/8ec037ce66ef584ccc66f85a6375f1721918b017/devices/ledvance.js#L171

**Reply by dhanukapulkit** [reply]
Done and integrated. Thanks a lot!
